build.yaml

```diff
@@ -630,6 +657,37 @@ steps:
      - default_ns
      - create_batch_tables_image
      - batch_database
+ - kind: deploy
+   name: delete_batch2_tables
+   namespace:
+     valueFrom: default_ns.name
+   config: batch2/delete-batch-tables-pod.yaml
```

Comment:
I think this shouldn't go in, but feel free to disagree.  Mainly, I want to control when my test database gets flushed.  For example, if I fix a bug and re-deploy, I don't want running batches to get blown away.  Sooner rather than later, we're going to have to support database version migration, which should fix this long term.

Reply:
It was useful when I wanted to redeploy batch2 to have the database clean. I think we should keep it, but maybe not make it a dependency of `deploy_batch2`. But then you'd have to run dev deploy twice or we need to take a list of multiple steps to run and their dependencies. That would make me ok with not having it run by default.